#! /bin/sh /usr/share/dpatch/dpatch-run
## 60_version.dpatch by Romain Francoise <rfrancoise@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix hping version.

@DPATCH@
diff -urNad hping3-3.a2.ds1/release.h /tmp/dpep.rHLwje/hping3-3.a2.ds1/release.h
--- hping3-3.a2.ds1/release.h	2004-04-10 01:38:56.000000000 +0200
+++ /tmp/dpep.rHLwje/hping3-3.a2.ds1/release.h	2005-03-12 13:42:01.861378440 +0100
@@ -11,7 +11,7 @@
 #ifndef _RELEASE_H
 #define _RELEASE_H
 
-#define RELEASE_VERSION "3.0.0-alpha-1"
+#define RELEASE_VERSION "3.0.0-alpha-2"
 #define RELEASE_DATE "$Id: release.h,v 1.4 2004/04/09 23:38:56 antirez Exp $"
 #define CONTACTS "<antirez@invece.org>"
 
